src/plugins/console/server/services/spec_definitions_service.ts

```diff
@@ -115,8 +115,8 @@ export class SpecDefinitionsService {
   }
 
   private loadJSONSpecInDir(dirname: string) {
-    const generatedFiles = glob.sync(join(dirname, 'generated', '*.json'));
-    const overrideFiles = glob.sync(join(dirname, 'overrides', '*.json'));
+    const generatedFiles = globby.sync(join(dirname, 'generated', '*.json'));
```

Comment:
https://github.com/elastic/kibana/blob/4fdc70bf8c3b729733415820b48248abbfb5ddf8/src/plugins/console/server/services/spec_definitions_service.ts#L106-L115

Reply:
Addressed path normalization in 3a7b34cb96278cd376578f9d7dab9b0210264c54